src/main/java/frc/robot/Robot.java

```diff
@@ -3,12 +3,21 @@
 import com.flash3388.flashlib.frc.robot.FrcRobotControl;
 import com.flash3388.flashlib.frc.robot.base.iterative.DelegatingFrcRobotControl;
 import com.flash3388.flashlib.frc.robot.base.iterative.IterativeFrcRobot;
+import com.flash3388.flashlib.hid.XboxController;
+import frc.robot.subsystems.Intake;
 
 public class Robot extends DelegatingFrcRobotControl implements IterativeFrcRobot {
+    private Intake intake;
+    private XboxController xbox;
 
-    public Robot(FrcRobotControl robotControl) {
+    public Robot(FrcRobotControl robotControl)
+    {
         super(robotControl);
+        this.intake = SystemFactory.createIntake();
+        this.xbox = getHidInterface().newXboxController(RobotMap.XBOX);
+
```

Comment:
Add in here : 
xbox.getButton(XboxButton.B).whenActive(new TakeOut(intake));
Don't use a button because the intake will not stop on it's own